src/sources/libfaadloader.cpp

```diff
@@ -40,7 +40,8 @@ LibLoader::LibLoader()
     // Using MacPorts ('sudo port install faad2' command):
     libnames << "/opt/local/lib/libfaad2.dylib";
 #else
-    libnames << "libfaad.so";
+    libnames << "libfaad.so.2"
+             << "libfaad.so";
```

Comment:
We should get rid of the platform-specific hacks and let QLibrary do its job. Just use `faad` and put the absolute paths for macOS in the list before that.

Reply:
see https://github.com/daschuer/mixxx/pull/66/commits/0a009044744335629347abbf026b66743555e12d

Reply:
This would only work, if all libraries on all targets following the naming convention Qt uses. 
I don't think this is the case here. 

Since this is a solution for a bug, lets merge it now.